src/python/pythonUtils.ts

```diff
@@ -144,6 +144,8 @@ export async function runPython(
     mode: 'binary',
     pythonPath,
     scriptPath: __dirname,
+    // When `inherit` is used, `import pdb; pdb.set_trace()` will work.
+    stdio: getEnvBool('PROMPTFOO_PYTHON_PDB_TRACING_ENABLED') ? 'inherit' : undefined,
```

Comment:
prefer a conditional assingnment so the  stdio  property does not exist on the object 

```suggestion
    ...(getEnvBool('PROMPTFOO_PYTHON_PDB_TRACING_ENABLED') && { stdio: 'inherit' }),
```